packages/tools-api/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@l2beat/tools-api",
+  "version": "0.1.0",
+  "private": true,
+  "main": "./build/data.json",
+  "types": "./index.d.ts",
+  "scripts": {
+    "build": "tsc",
+    "clean": "rm -rf build",
+    "format": "biome format .",
+    "format:fix": "biome format --write .",
+    "lint": "biome check --formatter-enabled=false .",
+    "lint:fix": "biome check --formatter-enabled=false --write .",
+    "start": "node -r source-map-support/register build/index.js",
```

Comment:
Running `start` locally does not work because `source-map-support` is not installed. I don't know if it's intendent

Reply:
Must be a weird leftover from something. I'll remove this.